### ISSUE TITLE
chroot issue

### ISSUE THREAD

**Issue by Mannadew** [guardianproject/lildebi#171]
I have an issue I get the error "cant execute 'su' no such file or directory." when i try to run linux on linux deploy. 
so I do 
ls data/data/ru.meefik.linuxdeploy/linux/mnt su - root -c /etc/init.d/myscript start
says tmp -mksh: root: not found what do i do now?

**Comment by godlessfather** [comment]
'su - root' doesn't do what I think you're wanting it to do. Have you tried
'su -c /etc/init.d/myscript start'?

**Comment by Mannadew** [comment]
K will try there, thanks.

**Comment by Mannadew** [comment]
yes i did it says unknown option -c and aborting. I also did chroot
data/data/linuxdeploy/linux mnt su -c /etc/init.d/myscript start and it
says system/bin/sh: chroot: not found. So do i change the directory of
linux?
